src/Symfony/Component/Intl/Resources/data/languages/sh.json

```diff
@@ -1,5 +1,5 @@
 {
-    "Version": "2.1.8.20",
+    "Version": "2.1.13.24",
     "Names": {
         "aa": "Afarski",
         "ab": "abhaski",
```

Comment:
It's better to review this PR commit by commit. "[Intl] Update the ICU data to version 55" is an update for generated files.